packages/cli/yaml/yaml-schema/src/schemas/ObjectPropertySchema.ts

```diff
@@ -1,6 +1,6 @@
 import { z } from "zod";
-import { TypeReferenceDeclarationWithNameSchema } from "./TypeReferenceSchema";
+import { TypeReferenceDeclarationWithContentType } from "./TypeReferenceSchema";
 
-export const ObjectPropertySchema = TypeReferenceDeclarationWithNameSchema;
```

Comment:
@abvthecity because we don't want to affect all object properties I would create a separate file called `InlinedObjectPropertySchema` and leverage the `TypeReferenceDeclarationWithContentType` type. 

Then `HttpInlineRequestBodySchema` can depend on the `InlinedObjectPropertySchema` instead.